guides/common/modules/snip_host-names-for-http-proxy.adoc

```diff
@@ -0,0 +1,14 @@
+[id="host-names-for-http-proxy_{context}"]
+
+[cols="2,1,1",options="header"]
+|====
+| Host name | Port | Protocol
+| subscription.rhsm.redhat.com | 443 | HTTPS
+| subscription.rhn.redhat.com | 443 | HTTPS
+| cdn.redhat.com | 443 | HTTPS
+| *.akamaiedge.net | 443 | HTTPS
+ifdef::satellite[]
+| api.access.redhat.com (if using Red{nbsp}Hat Insights) | 443 | HTTPS
```

Comment:
I think you accidentally lost the `cert.console.redhat.com` line

Reply:
Apologies for this. I assumed there is only `ifdef` difference in the table of both files. But, a few host differences were also there. Addressed :+1: